docs/Report.md

```diff
@@ -111,34 +88,31 @@ Then, confirmation of success is sent back until the razorpage redirects to itse
- Because it runs the tests as well, it ensures that any incoming changes do not affect the functionality. If anything fails, it will stop and prevent the branch from merging into main.
-<figure>
-   <center> <img src="Images/BuildAndTest.png"
-         alt="BuildAndTest Activity diagram">
-    <figcaption>Fig.XX Build and test activity diagram</figcaption></center>
-</figure>
+The build and test workflow can be found [here](#BuildTest). The activity diagram shows how GitHub ensures what is merged into main does not damage it. 
+This workflow is run on a pull request every time a commit is made to the branch belonging to the pull request. 
+This ensures that main will stay functional by building and testing the project with dotnet and our test suite. If anything fails, it will stop and prevent the branch from merging into main.
+
+[BuildTest]: Images/BuildAndTest.png "Build and test activity diagram" 
+![Build and test activity diagram][image]{width=60%}
+
+
 
 #### Publish and release
 
-This workflow is made to automate the creation of a GitHub release when a tag is added (Appendix?). It will create a release of the tag. But first, the workflow builds a version for Windows, MacOS and Linux. After that, it will zip the files and add them to the release if a release was made.
-<figure>
-   <center> <img src="Images/PublishNewRelease.png"
-         alt="PublishNewRelease Activity diagram">
-    <figcaption>Fig.XX Publish new release activity diagram</figcaption></center>
-</figure>
+This workflow is made to automate the creation of a GitHub release when a tag is added (Appendix?). It will create a release of the tag. But first, the workflow in succession builds a version for Windows, MacOS and Linux. After that, it will zip the files and add them to the release if a release was made.
+
+
+![Publish New Release Activity diagram](Images/PublishNewRelease.png){width=60%}
 
 #### Build and deploy
-This workflow can be seen here (Appendix?). The workflow is made so it will build the program and run the "publish" command to build a version for Linux to be run on the Azure web app. After the publish command, it uploads the artifacts so the next job can use the files. The deploy job will download the artefact and use the files to deploy to our Azure web app.
-<figure>
-   <center> <img src="Images/BuildAndDeploy.png"
-         alt="BuildAndDeploy Activity diagram">
-    <figcaption>Fig.XX Build and deploy activity diagram</figcaption></center>
-</figure>
+This workflow can be seen here (Appendix?). The workflow is made so it will build the program and run the "publish" command to build a version for Linux to be run on the Azure web app. After the publish command, it uploads the artefacts so the next job can use the files of the artefacts. The deploy job will download the artefacts and use their files to deploy to our Azure web app.
```

Comment:
Increased clarity around artefacts and made sure it was spelled similarly

Reply:
Looks good